modules/claims_api/app/models/claims_api/veteran.rb

```diff
@@ -69,13 +69,12 @@ def self.from_headers(headers, with_gender: false)
         first_name: ensure_header(headers, 'X-VA-First-Name'),
         last_name: ensure_header(headers, 'X-VA-Last-Name'),
         va_profile: build_profile(headers),
-        edipi: ensure_header(headers, 'X-VA-EDIPI'),
         last_signed_in: Time.now.utc
       )
       # commenting this out until the new non-veteran oauth flow is ready to replace this
       # veteran.loa = { current: 3, highest: 3 }
       veteran.gender = ensure_header(headers, 'X-VA-Gender') if with_gender
-
+      veteran.edipi = headers['X-VA-EDIPI'] if headers['X-VA-EDIPI'].present?
```

Comment:
I thought the goal here was to remove EDIPI as a necessary option, it looks more like you're requiring still and or implementing it in the MVI model?

Reply:
I am not requiring it which is why I am not calling ensure header, I am just setting it if passed. I figured that since with mvi the more info the better. We know we are getting data back for the users we have tested without edipi, but maybe there are use cases where MVI can't find the user and optionally sending up edipi would help. At least that was my thinking.

Reply:
Got it.